app/workers/clear_invalid_search_references.rb

```diff
@@ -0,0 +1,14 @@
+class ClearInvalidSearchReferences
+  include Sidekiq::Worker
+
+  sidekiq_options retry: false
+
+  def perform
+    SearchReference.where(referenced_class:'Heading').each do |ref|
```

Comment:
Not just headings, btw

Reply:
Good point, but strangely, when it is not filtered by heading, the execution of the query raises an error.
The ticket only reports the error for `Heading`, so I guess it's ok to release it only for Heading for now and then iterate to fix the other kind of entries.

I've already tested it and deleted the same nine entries mentioned in the ticket.

Reply:
👍 as discussed, to fix this error adding a migration to delete the search references to secions.